src/System.Memory/tests/Span/Reflection.cs

```diff
@@ -0,0 +1,224 @@
+
+            int result = (int)method.Invoke(null, new object[] { default, default, StringComparison.Ordinal });
+            Assert.Equal(0, result);
+        }
+
+        [Fact]
+        public static void MemoryExtensions_Contains()
+        {
+            Type type = typeof(MemoryExtensions);
+
+            MethodInfo method = type.GetMethod("Contains");
+
+            bool result = (bool)method.Invoke(null, new object[] { default, default, StringComparison.Ordinal });
+            Assert.True(result);
+        }
+
+        [Fact]
+        public static void MemoryExtensions_IsWhiteSpace()
+        {
+            Type type = typeof(MemoryExtensions);
+
+            MethodInfo method = type.GetMethod("IsWhiteSpace");
+
+            bool result = (bool)method.Invoke(null, new object[] { default });
+            Assert.True(result);
+        }
+
+        [Fact]
+        public static void MemoryExtensions_ToLower()
+        {
+            Type type = typeof(MemoryExtensions);
+
+            MethodInfo method = type.GetMethod("ToLower");
+
+            int result = (int)method.Invoke(null, new object[] { default, default, CultureInfo.InvariantCulture });
+            Assert.Equal(0, result);
+        }
+
+        [Fact]
+        public static void MemoryExtensions_ToLowerInvariant()
+        {
+            Type type = typeof(MemoryExtensions);
+
+            MethodInfo method = type.GetMethod("ToLowerInvariant");
+
+            int result = (int)method.Invoke(null, new object[] { default, default });
+            Assert.Equal(0, result);
+        }
+
+        [Fact]
+        public static void MemoryExtensions_ToUpper()
+        {
+            Type type = typeof(MemoryExtensions);
+
+            MethodInfo method = type.GetMethod("ToUpper");
+
+            int result = (int)method.Invoke(null, new object[] { default, default, CultureInfo.InvariantCulture });
+            Assert.Equal(0, result);
+        }
+
+        [Fact]
+        public static void MemoryExtensions_ToUpperInvariant()
+        {
+            Type type = typeof(MemoryExtensions);
+
+            MethodInfo method = type.GetMethod("ToUpperInvariant");
+
+            int result = (int)method.Invoke(null, new object[] { default, default });
+            Assert.Equal(0, result);
+        }
+
+        [Fact]
+        public static void Span_Ctor()
+        {
+            Type type = typeof(Span<int>);
+
+            ConstructorInfo ctor = type.GetConstructor(new Type[] { typeof(int[]) });
+            Assert.Throws<TargetException>(() => ctor.Invoke(new object[] { new int[10] }));
+
+            ctor = type.GetConstructor(new Type[] { typeof(int[]), typeof(int), typeof(int) });
+            Assert.Throws<TargetException>(() => ctor.Invoke(new object[] { new int[10], 1, 1 }));
+
+            ctor = type.GetConstructor(new Type[] { typeof(void*), typeof(int) });
+            Assert.Throws<TargetException>(() => ctor.Invoke(new object[] { null, 1 }));
```

Comment:
Anything to validate about the TargetException's inner exception?

Reply:
What should we validate about the inner exception? That it is null?

Reply:
If the inner exception is null, then nevermind.  I thought TargetException is often thrown from reflection when it's wrapping another exception, but maybe that's only true of TargetInvocationException?